### PR TITLE
Added Gincoin support

### PR DESCRIPTION
**Pull request by bedeabza**

### CODE REVIEW

File: defs/bitcoin/gincoin.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "Normal": 10
+  },
+  "dust_limit": 5460,
+  "blocktime_seconds": 120,
+  "uri_prefix": "gincoin",
+  "min_address_length": 27,
+  "max_address_length": 34,
+  "bitcore": [],
+  "blockbook": [
+    "https://blockbook.gincoin.io/"
```
Comment (prusnak) [inline defs/bitcoin/gincoin.json]: remove the trailing '/'

Comment (bedeabza) [reply defs/bitcoin/gincoin.json]: Done

File: defs/bitcoin/gincoin.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  "coin_label": "GIN",
+  "website": "https://gincoin.io",
+  "github": "https://github.com/gincoin-dev/gincoin-core",
+  "maintainer": "Dragos Badea <dragos.badea@gincoin.io>",
+  "curve_name": "secp256k1",
+  "address_type": 38,
+  "address_type_p2sh": 10,
+  "maxfee_kb": 100000,
+  "minfee_kb": 1000,
+  "signed_message_header": "DarkCoin Signed Message:\n",
```
Comment (prusnak) [inline defs/bitcoin/gincoin.json]: Are you sure Gincoin developers were sloppy and did not change this constant?

Comment (bedeabza) [reply defs/bitcoin/gincoin.json]: They were indeed

**Pull request merged**